RocketChat/Rocket.Chat
# fix: First response time livechat metrics are associated with the last agent who served the room

## Proposed changes (including videos or screenshots)

 - Fixed rooms' "first response time" metric being attached to the last agent who served a room instead of the first one -- this leads to incorrect values for the "Best first response time" and "Average first response time" metrics for each agent in case rooms are forwarded among agents/departments.

## Issue(s)


## Steps to test or reproduce

Make sure to grant the `transfer-livechat-guest` permission to agents or to a manager. 
Create a new agent and respond a livechat room with it, then transfer the room to another one. The "first response time" aggregation metrics are available under **Omnichannel > Analytics > Productivity** (or by using the `livechat/analytics/agent-overview` endpoint), and we need to make sure that both metrics related to the room that has been transferred are associated to the first agent who responded to the room (and not to the last one who served it).

## Further comments

[CORE-669]

[CORE-669]: https://rocketchat.atlassian.net/browse/CORE-669?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
